src/thunderbots/software/ai/hl/stp/stp.cpp

```diff
@@ -238,7 +239,7 @@ std::optional<std::string> STP::getCurrentPlayName() const
 PlayInfo STP::getPlayInfo()
```

Comment:
Doesn't look like we have test coverage for this function, a quick test here would at least sanity check it, and also the `name` function below.

Reply:
added unit tests